ignition-server/controllers/machineconfigserver_ignitionprovider.go

```diff
@@ -186,7 +211,8 @@ name="${1}"
 oc get cm ${name} -n "${NAMESPACE}" -o jsonpath='{ .data.data }' > "/mcc-manifests/bootstrap/manifests/${name/#ignition-config-//}.yaml"
 EOF
 chmod +x ./copy-ignition-config.sh
-oc get cm -l ignition-config="true" -n "${NAMESPACE}" --no-headers | awk '{ print $1 }' | xargs -n1 ./copy-ignition-config.sh`
+oc get cm -l ignition-config="true" -n "${NAMESPACE}" --no-headers | awk '{ print $1 }' | xargs -n1 ./copy-ignition-config.sh
```

Comment:
This is unimportant on this PR but just a thing I was thinking of:
Do we need something like a `set -o pipefail` to ensure temporary network errors on the ignition config fetch don't silently get squashed?

Reply:
Yup, that makes sense. If not addressed in this PR we should do a follow up.

Reply:
Make sense, updated.